plenum/test/view_change_with_delays/test_view_change_with_tnx_ordered_on_only_one_node_and_restart_of_others.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from plenum.test.helper import perf_monitor_disabled
+from plenum.test.view_change_with_delays.helper import do_view_change_with_delayed_commits_and_node_restarts
+
+
+@pytest.fixture(scope="module")
+def tconf(tconf):
+    """
+    Patch config so that monitor won't start view change unexpectedly
+    """
+    with perf_monitor_disabled(tconf):
+        yield tconf
+
+
+"""
+    This tests exhibits weird behaviour that should be investigated.
```

Comment:
I would suggest calling `sdk_get_and_check_replies ` in `do_view_change_with_delayed_commits_and_node_restarts`, it may help.
In general the test should fail now and will be fixed by INDY-2238